source/agora/consensus/protocol/Nominator.d

```diff
@@ -399,29 +399,29 @@ extern(D):
 
         Params:
             slot_idx = the slot index we're currently reaching consensus for.
-            timer_ID = the timer ID. required in case the timer gets cancelled.
+            timer_type = the timer type (see Slot.timerIDs).
```

Comment:
Uuuurgh, so there's only 2 timers ? Why didn't just make 2 functions ...

Reply:
I guess maybe they plan on making more?

Can we just delete SCP?

Reply:
> Can we just delete SCP?

I mean... Now that we got most of the D compiler bugs fixed, it would be a shame.